cli/command/environment_command.go

```diff
@@ -46,6 +46,10 @@ func (e *EnvironmentCommand) GetCommand() cli.Command {
 						Value: "linux",
 						Usage: "specifies windows or linux container environment cluster",
 					},
+					cli.StringFlag{
+						Name:  "ami",
+						Usage: "specifies a custom AMI to use in the environment",
```

Comment:
agreed w tlake, we should put an example of the AMI ID format, too just to be on the safe side